packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -153,6 +153,12 @@ class CommandHelp {
     'Toggle elevation checker.',
   );
 
+  CommandHelpOption _k;
```

Comment:
Should this check if we're running a web build?

Reply:
Also, will this same switch work for non-debug builds?

Reply:
See https://github.com/flutter/flutter/pull/52511/files#diff-62dbce17da9d1284c7f14ae07e67df33R668 for the web build guarding

And https://github.com/flutter/flutter/pull/52511/files#diff-4951af41b5bd06e6f3f90860ba6029daR714 for the debug mode check.


This won't work in release mode, since we need to build from source in those